api/src/models/JobMetadata.ts

```diff
@@ -43,6 +43,10 @@ class JobMetadata {
         return job.dir + '/ingest.lock';
     }
 
+    validate(job, params) {
+
```

Comment:
We can talk about how validation works next week; I've refreshed my memory, and it is indeed a bit complicated. It might make sense to use a separate validator class rather than building it in... or it might not. We can discuss next week!